<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maintenancePage</class>
 <widget class="QWidget" name="maintenancePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="product4_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>680</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="product1_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="product3_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>1040</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="product2_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>590</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="product2_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="product3_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>680</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="product4_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>1040</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="product1_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>590</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>1220</y>
     <width>271</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>691</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MAINTENANCE MODE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wifiButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>1670</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>WiFi</string>
   </property>
  </widget>
  <widget class="QLabel" name="wifi_status">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>1730</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Wifi Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="wifi_internet">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>1800</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Internet connection</string>
   </property>
  </widget>
  <widget class="QLabel" name="wifi_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>1670</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Wifi Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
